clusters/build-clusters/01_cluster/openshift/scc/README.md

```diff
@@ -0,0 +1,3 @@
+# security context constraints
+
+This folder contains the SCCs that we modified.
```

Comment:
This is wrong, the SCC that we _added_ or not?

Reply:
Updated.